docs/getting_started/requirements.md

```diff
@@ -111,3 +111,77 @@ These setups have been undergone some additional manual testing and is known to
 -   Opera® most recent stable version, or higher, desktop *and* mobile 
 
 Please note that the user interface might not look or behave exactly the same across all browsers as it will gracefully degrade if browser does not support certain features.
+
+
+## eZ Platform Cloud requirements and setup
+
+### Cloud hosting with eZ Platform Cloud and Platform.sh
+
+In general, eZ Platform Cloud supports all features and services of [Platform.sh](https://platform.sh/hosting/php/ez) that are compatible and supported by the eZ Platform version you use.  
+
+For example:
+
+- Platform.sh provides Redis support for versions 2.8, 3.0 and 3.2. eZ Platform supports Redis version 3.0 or higher. As a result, Redis is supported by eZ Platform Cloud versions 3.0 and 3.2.
+- Platform.sh provides Elastic Search service (different versions supported). eZ Platform 1.13 *does not* officially support Elastic Search. As a result, Elastic Search is not supported for use as search engine for eZ Platform (1.13).
+
+Features or services supported by eZ Platform but not covered by Platform.sh may be possible by means of a [custom integration](#custom-integrations).
+
+### eZ Platform Cloud Setup support matrix
+
+|Setup|Description|eZ Platform Cloud support|
+|-----|-----|-----|
+|Recommended setup|**eZ Platform**</br>eZ Platform setup, with no use of legacy.|eZ Platform software plus cloud hosting infrastructure supported by eZ for version 1.13 and higher</br></br>Recommended configuration provided out of the box and fully supported, on-boarding help available.|
+|Supported setup|**Legacy Bridge setup**</br>eZ Publish 5.x-like setup where web traffic goes to eZ Platform/Symfony and Legacy Bundle provides legacy fallback features.</br>Notably it allows among other things enabling a "Legacy mode" where legacy bundle lets legacy take over handling of URL aliases.|Installation and cloud supported by eZ for version 1.13 and 2.x</br></br>Review and potential adaptation of the application configuration to be performed by eZ Systems technical services prior to deployments. The cost and effort of this review and potential adaptation is not included in eZ Platform Cloud subscription and will vary depending on each project's specificities.|
+|Experimental setup|**eZ Publish, eZ Publish Platform and other pure legacy setups**</br>eZ Publish (version 4.x) or eZ Publish Platform (version 5.x) standalone setup or an experimental setup with eZ Platform and Legacy Bridge.|Not covered by eZ Platform Cloud subscription.</br></br>The use of Platform.sh service is possible as a standalone service and can be used to bridge migration needs. Strong involvement of an eZ business partner with existing knowledge around setup of legacy projects on Platform.sh strongly recommended|
```

Comment:
**Involvement** of an eZ business partner **that has experience with setting up** legacy projects on Platform.sh **is strongly recommended**.

OR

We recommend involving eZ business partner that has experience with setting up legacy projects on Platform.sh.